requirements.in

```diff
@@ -12,3 +12,4 @@ requests_kerberos
 requests
 semver
 setuptools
+SQLAlchemy<1.4
```

Comment:
This should be in `requirements-web.in` (it is a dependency of `Flask-SQLAlchemy`)